### PR TITLE
Create v1 changelog

### PR DESCRIPTION
**Pull request by youngnick**
/kind documentation

Adds the v1 changelog.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,183 @@
...
+This release marks a huge milestone for this project. Several key APIs are
+graduating to GA (generally available), while other significant features have
+been added to the Experimental channel.
+
+It's been four years since this project began, and we would never have got here
+without the support of a dedicated and active community. The maintainers would like
+to thanks _everyone_ who's contributed to Gateway API, whether in the form of
+commits to the repo, discussion, ideas, or general support.
+
+We literally couldn't have got this far without you.
```
Comment (shaneutt) [inline CHANGELOG.md:47]: The bottom sentence correlates to the paragraph.

```suggestion
commits to the repo, discussion, ideas, or general support. We literally couldn't have got this far without you.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -32,6 +32,183 @@
...
+
+This project is nowhere near finished, as you can see from the large amount of
+features being added into Experimental.
+
+With such a big set of things still to do, contributors and contributions are
```
Comment (shaneutt) [inline CHANGELOG.md:52]:
```suggestion
features being added into Experimental. With such a big set of things still to do,
contributors and contributions are
```

File: CHANGELOG.md (hunk 1 above)
Comment (robscott) [reply CHANGELOG.md:47]: One additional nit - I think "got" should be "gotten" here.

File: CHANGELOG.md (hunk 2 above)
Comment (robscott) [reply CHANGELOG.md:52]: Maybe say "Experimental Channel" just to be extra clear for people new to the project.

**Review by robscott — COMMENTED** [review]
Thanks @youngnick! One small request - do you mind also bumping the version numbers from v1.0.0-rc2 and v0.8.1 to v1.0.0 in this PR?

**Review by shaneutt — APPROVED** [review]
I think we're good to go now

/approve
/lgtm

/hold for you @robscott to give one last check

**Review by shaneutt — APPROVED** [review]
/approve
/lgtm

**Pull request merged**